package.json

```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "pretest": "abaplint --version && abaplint \"src/**/*.abap\"",
     "test": "abapmerge src/zabapgit.prog.abap > zabapgit.abap",
-    "_disabled_posttest": "abaplint zabapgit.abap"
+    "_disabled_posttest": "abaplint zabapgit.abap",
+    "lint": "node_modules/.bin/abaplint \"src/**/*.abap\""
```

Comment:
we'll probably have to change this part sometime later

Reply:
IMHO:
- the lint script is OK, it is even typical
- but I'd change the travis script instead: lint and merge is done indirectly at the moment, this is confusing and not transparent (although this is not most critical AG part :)))). 
- I'd suggest splitting travis script into stages, one for pull requests, another for merge. Then there can be scripts like "lint", "merge/build", and the stage committing the build artifacts. And ... I'd better remove private key from travis.yml and regenerate it too ... :)